docs/observatorium/loki/README.md

```diff
@@ -0,0 +1,52 @@
+## Loki Access
+
+Loki access is open to public and is divided into 3 different components:
+1. Loki Distributor
+2. Loki Frontend
+3. Grafana
+
+While pushing logs into Loki or while querying Loki,
+you will need to have a header `'X-Scope-OrgID'` set
+in your requests. <br>
+The value for this header can be anything.
+
+For our examples, we will set it to `'X-Scope-OrgID: opf-example'`.
+
+### Loki Distributor
+* url: http://loki-distributor-opf-observatorium.apps.cnv.massopen.cloud/
+
+This is the endpoint that can be used to push logs into Loki. <br>
+
+Example:
+```bash
+curl -v -H "Content-Type: application/json" \
+  -H "X-Scope-OrgID: opf-example" \
+  -XPOST -s "http://loki-distributor-opf-observatorium.apps.cnv.massopen.cloud/loki/api/v1/push" --data-raw \
+  '{"streams": [{ "stream": { "app": "my-app-1" }, "values": [ [ "1607526347000000000", "my-log-line" ] ] }]}'
+```
+More Info on  [/push](https://grafana.com/docs/loki/latest/api/#post-lokiapiv1push)
+
+### Loki Frontend
+* url: http://loki-frontend-opf-observatorium.apps.cnv.massopen.cloud/
+
+This is the endpoint that can be used to query logs from Loki. <br>
+
+Example:
+```bash
+curl -H "X-Scope-OrgID: opf-example" \
+  -G -s  "http://loki-frontend-opf-observatorium.apps.cnv.massopen.cloud/loki/api/v1/query_range" \
+  --data-urlencode 'query={app="my-app-1"}' \
+  --data-urlencode 'start=1607526347000000000'
+```
+More Info on [/query_range](https://grafana.com/docs/loki/latest/api/#get-lokiapiv1query_range)
```

Comment:
Both the push/query endpoints work great. I think it's great that you link to the docs for additional info, but maybe you can add quick one liner describing what exactly the curl commands are doing.

Example: "Here we are querying the orgid opf-example for logs that we entered earlier belonging to the app "my-app-1"

Reply:
should be updated now :+1: